examples/language/bert/README.md

```diff
@@ -0,0 +1,34 @@
+## Overview
+
+This directory includes two parts: Using the Booster API fintune Huggingface Bert and AlBert models and benchmarking Bert and AlBert models with different Booster Plugin.
+
+## Finetune
+```
+bash test_ci.sh
+```
+
+## Benchmark
+```
+bash benchmark.sh
+```
+
+Now include these metrics in benchmark: CUDA mem occupy，throughput and the number of model parameters. If you have a custom metrics, you can add it to benchmark_util.
```

Comment:
It seems that the first comma is the comma in Chinese. And this sentence has syntax error. You can use some tools like https://www.nounplus.net/grammarcheck/ to check your sentence.